### PR TITLE
Angular88

### PR DESCRIPTION
**Pull request by vrkansagara**

### CODE REVIEW

**Comment by vrkansagara** [comment]
merge pr

**Comment by anteburazer** [comment]
@vrkansagara Thank you for the contribution. This repo is direct reflection of series of blog posts at [http://netmedia.io](https://netmedia.io/dev/angular-architecture-patterns-high-level-project-architecture_5589). I can see that you didn't just update the Angular version but you changed the project structure and added more logic which could be misleading with the articles.

I'm not maintaining this repo anymore but I suggest that we create two PRs against different branches:

1. Angular 8 update (this one can go to master directly)
2. New logic that you introduced. You can put a summary about what was your idea and what changes you have introduced so it can be added into README.md and point to different branch. In this case the articles would be in sync with master branch and readers could explore more in separate branch.

cc @ivanstrigo

File: src/app/shared/services/utils.service.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import { HttpRequest } from '@angular/common/http';
+import { Observable } from 'rxjs/';
+import { createBody } from '../asyncServices/http/createBody';
+import { createHeaders } from '../asyncServices/http/createHeaders';
+import { createPath } from '../asyncServices/http/createPath';
+import { createQuery } from '../asyncServices/http/createQuery';
```
Comment (danielsefton) [inline src/app/shared/services/utils.service.ts:6]: Where are these files?

File: src/app/shared/services/utils.service.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      const pPath = target[`${propertyKey}_Path_parameters`];
+      const pQuery = target[`${propertyKey}_Query_parameters`];
+      const pBody = target[`${propertyKey}_Body_parameters`];
+      const pHeader = target[`${propertyKey}_Header_parameters`];
+
+      descriptor.value = function (...args: any[]) {
+        const body: string = createBody(pBody, descriptor, args);
+        const resUrl: string = createPath(url, pPath, args);
+        const search: URLSearchParams = createQuery(pQuery, args);
+        const headers: Headers = createHeaders(pHeader, descriptor, this.getDefaultHeaders(), args);
```
Comment (danielsefton) [inline src/app/shared/services/utils.service.ts:23]: search and headers vars are unused?

**Pull request closed without merging**